app/src/lib/utils/selectFilesInList.ts

```diff
@@ -1,24 +1,24 @@
-import { fileKey, type FileSelection } from '$lib/vbranches/fileSelection';
+import { fileKey, type FileIdSelection } from '$lib/vbranches/fileIdSelection';
 import { get } from 'svelte/store';
 import type { AnyCommit, AnyFile } from '$lib/vbranches/types';
 
 export function selectFilesInList(
 	e: MouseEvent,
 	file: AnyFile,
-	fileSelection: FileSelection,
+	fileIdSelection: FileIdSelection,
 	sortedFiles: AnyFile[],
 	allowMultiple: boolean,
 	commit: AnyCommit | undefined
 ) {
-	let selectedFileIds = get(fileSelection);
+	let selectedFileIds = get(fileIdSelection);
 	e.stopPropagation();
-	const isAlreadySelected = selectedFileIds && fileSelection.has(file.id, commit?.id);
+	const isAlreadySelected = selectedFileIds && fileIdSelection.has(file.id, commit?.id);
```

Comment:
Let's simplify since `selectedFiles` is never undefined. I likely introduced it, but it's unnecessary.